frida_tools/repl.py

```diff
@@ -629,6 +644,33 @@ def _make_repl_runtime(self, indent):
 global.cm = null;
 global.cs = {};
 
+class REPL {
+    #cmdDispatcher;
+    constructor() {
+        this.#cmdDispatcher = new Map();
+    }
+    _doQuickCmd(s) {
+        const args = JSON.parse(decodeURIComponent(s));
+        const cmd = args[0];
+        const func_info = this.#cmdDispatcher.get(cmd)
```

Comment:
```suggestion
        const funcInfo = this.#cmdDispatcher.get(cmd)
```

(Here and elsewhere:) Variables should always be `camelCase`.

Reply:
Modified with RPC using and camelCase used.